### PR TITLE
Setup proper path for node_agent

### PR DESCRIPTION
**Pull request by wattli**
**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: cmd/node_agent/main.go [hunk 1]
```diff
@@ -39,14 +39,14 @@ func init() {
 
 	flags := rootCmd.Flags()
 
-	flags.StringVar(&naConfig.ServiceIdentity, "service-identity", "", "Service Identity the node agent is managing")
-	flags.StringVar(&naConfig.ServiceIdentityOrg, "org", "Juju org", "Organization for the cert")
+	flags.StringVar(&naConfig.ServiceIdentity, "service-identity", "default", "Service Identity the node agent is managing")
```
Comment (lookuptable) [inline cmd/node_agent/main.go]: Is this a reasonable default? I don't issue any identity called "default".

Comment (wattli) [reply cmd/node_agent/main.go]: in the beginning I thought it's the right one, updated.

File: cmd/node_agent/main.go [hunk 2]
```diff
@@ -39,14 +39,20 @@ func init() {
...
-	flags.StringVar(&naConfig.NodeIdentityCertFile, "na-cert", "", "Node Agent identity cert file")
-	flags.StringVar(&naConfig.NodeIdentityPrivateKeyFile, "na-key", "", "Node identity private key file")
-	flags.StringVar(&naConfig.IstioCAAddress, "ca-address", "127.0.0.1", "Istio CA address")
+	flags.StringVar(&naConfig.NodeIdentityCertFile, "na-cert",
+		"/etc/certs/na-cert.pem", "Node Agent identity cert file")
+	flags.StringVar(&naConfig.NodeIdentityPrivateKeyFile,
+		"na-key", "/etc/certs/na-key.pem", "Node identity private key file")
+	flags.StringVar(&naConfig.IstioCAAddress,
+		"ca-address", "istio-ca.default.svc.cluster.local:8060", "Istio CA address")
 	flags.StringVar(&naConfig.ServiceIdentityDir, "cert-dir", "./", "Certificate directory")
```
Comment (myidpt) [inline cmd/node_agent/main.go]: Default /etc/certs/ ?

File: cmd/node_agent/na/nodeagent.go [hunk 3]
```diff
@@ -204,10 +204,10 @@ func (na *nodeAgentInternal) getWaitTimeFromCert(
 
 func (na *nodeAgentInternal) writeToFile(privKey []byte, cert []byte) error {
 	glog.Infof("Write key and cert to local file.")
-	if err := ioutil.WriteFile("serviceIdentityKey.pem", privKey, 0600); err != nil {
+	if err := ioutil.WriteFile("/etc/certs/key.pem", privKey, 0600); err != nil {
```
Comment (myidpt) [inline cmd/node_agent/na/nodeagent.go]: Concatenate ServiceIdentityDir with "Key.pem"?

File: cmd/node_agent/main.go (hunk 2 above)
Comment (wattli) [reply cmd/node_agent/main.go]: I will remove this flag, we don't need two flags for one file, and that's less readable too.

File: cmd/node_agent/na/nodeagent.go (hunk 3 above)
Comment (wattli) [reply cmd/node_agent/na/nodeagent.go]: As above

File: cmd/node_agent/main.go (hunk 2 above)
Comment (myidpt) [reply cmd/node_agent/main.go]: Sure.

**Pull request merged**